app/operations/kinesis/count_classification.rb

```diff
@@ -11,6 +11,7 @@ class CountClassification < Operation
         array :user_group_ids, default: [] do
           integer
         end
+        string :user_group, default: ""
```

Comment:
Shouldn't this be "user_group_id", or preferably even something longer to indicate that it's a group actively chosen by the user?

Reply:
Maybe. It’s up to @srallen, I’m using the name she gave the param in PFE.

Reply:
Yeah, I named it `user_group` in PFE. I could rename it, but would need to coordinate with Amy on updating the caesar configs too.

Reply:
@marten Is this a blocking problem?

Reply:
Right now it's still possible to change it right? Might as well do our best to make naming not too confusing while we still can.

Reply:
Do you have something more specific in mind? `user_group_id` right next to `user_group_ids` is just as confusing. We considered `classroom` except that's not accurate (it's a user group, not a classroom id). Right now, that doesn't seem better.

Regardless, it'll have to be changed everywhere else for this to matter but changing it here first won't break anything.

Reply:
The site has only been used for demos and trainings so far, so it's probably ok to change. How about `selected_user_group_id`?

Reply:
👍